spec/integration/application/lookup_spec.rb

```diff
@@ -107,7 +110,9 @@ def explain(key, options = {})
       require 'puppet/indirector/node/exec'
       require 'puppet/indirector/node/plain'
 
-      let(:node) { Puppet::Node.new('testnode', :environment => env) }
+
+      let(:facts) { Puppet::Node::Facts.new("facts", {}) }
+      let(:node) { Puppet::Node.new('testnode', :facts => facts, :environment => env) }
```

Comment:
These are not needed since the variables and their contents are identical to the ones at line `48/49`.